traffic_ops/bin/traffic_ops_ort.pl

```diff
@@ -646,18 +713,28 @@ sub update_trops {
 			}
 		}
 		elsif ( $script_mode == $BADASS || $script_mode == $SYNCDS ) {
-			&send_update_to_trops($CLEAR);
+			if ( defined $reval_pending ) {
+				&send_update_to_trops($CLEAR, $reval_pending );
+			}
+			else {
+				&send_update_to_trops($CLEAR, $CLEAR);
+			}
+		}
+		elsif ( $script_mode = $REVALIDATE ) {
+			&send_update_to_trops($upd_pending, $CLEAR);
 		}
 	}
 }
 
 sub send_update_to_trops {
 	my $status = shift;
-	my $url    = "$traffic_ops_host\/update/$hostname_short";
+	my $reval_status = shift;
+	my $uri    = "/update/$hostname_short";
 	( $log_level >> $DEBUG ) && print "DEBUG Setting update flag in Traffic Ops to $status.\n";
 
 	my %headers = ( 'Cookie' => $cookie );
-	my $response = $lwp_conn->post( $url, [ 'updated' => $status ], %headers );
+	my $url = $traffic_ops_host . $uri;
+	my $response = $lwp_conn->post( $url, [ 'updated' => $status, 'reval_updated' => $reval_status ], %headers );
```

Comment:
is it ok if reval_status is null here? Didnt trace the code, I just assume that can happen.